nextflow_schema.json

```
@@ -0,0 +1,498 @@
+                },
+                "longreads_keep_percent": {
+                    "type": "integer",
+                    "default": 90,
+                    "description": "Keep this percent of bases."
+                },
+                "longreads_length_weight": {
+                    "type": "integer",
+                    "default": 10,
+                    "description": "The higher the more important is read length when choosing the best reads.",
+                    "help_text": "The default value focuses on length instead of quality to improve assembly size.\nIn order to assign equal weights to read lengths and read qualities set this parameter to 1.\nThis might be useful, for example, to benefit indirectly from the removal of short host reads (causing lower qualities for reads not overlapping filtered short reads)."
+                },
+                "keep_lambda": {
+                    "type": "boolean",
+                    "description": "Keep reads similar to the ONT internal standard Escherichia virus Lambda genome."
+                },
+                "lambda_reference": {
+                    "type": "string",
+                    "default": "${baseDir}/assets/data/GCA_000840245.1_ViralProj14204_genomic.fna.gz",
+                    "hidden": true,
+                    "description": "Genome reference used to remove ONT Lambda contaminant reads."
+                }
+            }
+        },
+        "taxonomic_profiling_options": {
+            "title": "Taxonomic profiling options",
+            "type": "object",
+            "description": "Taxonomic classification is disabled by default. You have to specify one of the options below to activate it.",
+            "default": "",
+            "properties": {
+                "centrifuge_db": {
+                    "type": "string",
+                    "description": "Database for taxonomic binning with centrifuge.",
+                    "help_text": "E.g. \"ftp://ftp.ccb.jhu.edu/pub/infphilo/centrifuge/data/p_compressed+h+v.tar.gz\"."
+                },
+                "kraken2_db": {
+                    "type": "string",
+                    "description": "Database for taxonomic binning with kraken2.",
+                    "help_text": "E.g. \"ftp://ftp.ccb.jhu.edu/pub/data/kraken2_dbs/minikraken_8GB_202003.tgz\"."
+                },
+                "skip_krona": {
+                    "type": "boolean",
+                    "description": "Skip creating a krona plot for taxonomic binning."
+                },
+                "cat_db": {
+                    "type": "string",
+                    "description": "Database for taxonomic classification of metagenome assembled genomes.",
+                    "help_text": "E.g. \"http://tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20200618.tar.gz\".\nThe zipped file needs to contain a folder named \"*taxonomy*\" and \"*CAT_database*\" that hold the respective files."
```

CAT_prepare_20200618.tar.gz might not work, as discussed in https://github.com/nf-core/mag/issues/90. Instead, CAT_prepare_20200304.tar.gz works.
```suggestion
                    "help_text": "E.g. \"http://tbb.bio.uu.nl/bastiaan/CAT_prepare/CAT_prepare_20200304.tar.gz\".\nThe zipped file needs to contain a folder named \"*taxonomy*\" and \"*CAT_database*\" that hold the respective files."
```
Also, the help text looks partially italic, this might need polishing, see https://nf-co.re/mag/usage#--cat_db

---

hm... https://nf-co.re/mag/usage#--cat_db shows the last release version

---

Ah I see, still, CAT_prepare_20200618.tar.gz seems not perfect, is it?